src/Cli/Microsoft.TemplateEngine.Cli/Commands/create/InstantiateCommand.Help.cs

```diff
@@ -455,6 +455,7 @@ private static void WriteCustomInstantiateHelp(HelpContext context, CliCommand c
             HelpBuilder.Default.CommandArgumentsSection()(context);
             context.Output.WriteLine();
             HelpBuilder.Default.OptionsSection()(context);
+            context.Output.WriteLine();
```

Comment:
It did. The help output logic was changed and normalized. As you can see, previously, every other section here had an added `WriteLine` added to it *except* this one. It was likely being added (by accident) in the previous versions of S.CL.

I had to add it here so that the tests for checking help output passed and remained the same was as expected.